code/modules/power/pacman2.dm

```diff
@@ -30,16 +30,9 @@
 		P.air_contents.phoron -= 0.01
 		return
 
-	New()
-		..()
-		component_parts = list()
-		component_parts += new /obj/item/weapon/stock_parts/matter_bin(src)
-		component_parts += new /obj/item/weapon/stock_parts/micro_laser(src)
-		component_parts += new /obj/item/stack/cable_coil(src)
-		component_parts += new /obj/item/stack/cable_coil(src)
-		component_parts += new /obj/item/weapon/stock_parts/capacitor(src)
-		component_parts += new board_path(src)
-		RefreshParts()
+	Initialize()
```

Comment:
pathing

Reply:
For sure. But I can also totally understand keeping that separate given how it would make the diff be the entire file.

Reply:
Ye, that's why I did it. I think pathing fixes are generally their own thing. I definitely think it should be fixed, but maybe soooomeone should just go through and fix ALL the relative pathing at once.

Reply:
The problem with that, even moreso than replacing << with to_chat, is that there isn't a particularly easy or simple way of finding _all_ instances